docs/framework/windows-workflow-foundation/whats-new-in-wf-in-dotnet.md

```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```

Comment:
Was it a BOM or something?

Reply:
Ah, didn't see the commit message

Reply:
Yep 😄 I've asked them to change the error message in these cases to be more explicit. Not sure when it's going to be implemented.

Reply:
Soonish maybe https://github.com/MicrosoftDocs/azure-docs/issues/14166#issuecomment-459082416

This is how I found to clean it up before https://github.com/MicrosoftDocs/azure-docs/pull/10954#issue-198220272